microk8s-resources/wrappers/microk8s-reset.wrapper

```diff
@@ -17,6 +17,15 @@ Options:
 EOF
 }
 
+disable_addons() {
+  echo "Disabling all addons."
+  actions="$(get_all_addons)"
+  for action in $actions; do
+    echo "Disabling addon : $action"
+    microk8s.disable $action > /dev/null 2>&1 || true
```

Comment:
After enabling storage its disable process gets stuck in:
```
Remove PVC storage at /var/snap/microk8s/common/default-storage ? (Y/N)
```

Having a non interactive enable/disable mode for all addons would have made our life easier.

Reply:
I will work on this one. Thanks for reviewing.

Reply:
@ktsakalozos will it be fine to force an argument to `microk8s.disable storage:destroy-storage`  without specifying the argument will assume to keep the storage. wdyt?
thanks.

Reply:
Sorry I missed that @balchua. Yes this is a reasonable approach.

Reply:
@balchua I was wrong to suggest we could change the UI of `microk8s.disable storage` mid-cycle. On the last commit you can see I maintained the old behavior of asking for confirmation in deleteing the storage *if* no arguments were passed.